.github/workflows/post-merge-validation.yml

```
@@ -0,0 +1,283 @@
+      # Public tunnel exposure in CI is a security risk
+
+      - name: 🧪 Run Streamlined Tests
+        id: tests
+        env:
+          TEST_BASE_URL: "http://localhost:${{ env.CI_PORT }}"
+          GITHUB_EVENT_NAME: ${{ github.event_name }}
+          GITHUB_REF: ${{ github.ref }}
+        run: |
+          echo "Running tests against server at localhost:$CI_PORT"
+          echo "Also accessible via: ${NGROK_URL:-localhost}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)